<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1047</width>
    <height>690</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1047</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestione Corsi</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>690</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>GestisciCorsi.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="button_inserisciCorso">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>INSERISCI CORSO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_eliminaCorso">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>ELIMINA CORSO</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>621</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="list_view"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="GestisciCorsi.qrc"/>
 </resources>
 <connections/>
</ui>
